API.yaml

```diff
@@ -1,342 +1,419 @@
+          maximum: 4096
+          minimum: 0
+          type: integer
+          example: 4
+    ExposureInfo:
+      required:
+      - attenuation_durations
+      - attenuation_value
+      - date
+      - duration
+      - total_risk_score
+      - transmission_risk_level
+      type: object
+      properties:
+        date:
+          type: string
+          format: date
+        duration:
+          maximum: 3E+1
+          minimum: 5
+          type: integer
+          format: int32
+          example: 5
+        attenuation_value:
+          type: integer
+          format: int32
+          example: 45
+        attenuation_durations:
+          maxItems: 2
+          minItems: 2
+          type: array
+          example:
+          - 300
+          - 0
+          items:
+            minimum: 0
+            type: integer
+            format: int32
+        transmission_risk_level:
+          maximum: 8
+          minimum: 1
+          type: integer
+          format: int32
+          example: 1
+        total_risk_score:
+          maximum: 4096
           minimum: 0
-      maximum_risk_score:
-        type: "integer"
-        minimum: 0
-        maximum: 4096
-        example: 4
-  ExposureInfo:
-    type: "object"
-    required:
-    - "date"
-    - "duration"
-    - "attenuation_value"
-    - "attenuation_durations"
-    - "transmission_risk_level"
-    - "total_risk_score"
-    properties:
-      date:
-        type: "string"
-        format: "date"
-      duration:
-        type: "integer"
-        format: "int32"
-        maximum: 30
-        minimum: 5
-        example: 5
-      attenuation_value:
-        type: "integer"
-        format: "int32"
-        example: 45
-      attenuation_durations:
-        type: "array"
-        minItems: 2
-        maxItems: 2
-        example: [300, 0]
-        items:
-          type: "integer"
-          format: "int32"
+          type: integer
+          format: int32
+          example: 4
+    APIError:
+      required:
+      - error_code
+      - message
+      type: object
+      properties:
+        error_code:
           minimum: 0
-      transmission_risk_level:
-        type: "integer"
-        format: "int32"
-        minimum: 1
-        maximum: 8
-        example: 1
-      total_risk_score:
-        type: "integer"
-        format: "int32"
-        minimum: 0
-        maximum: 4096
-        example: 4
-  APIError:
-    type: "object"
-    required:
-    - "error_code"
-    - "message"
-    properties:
-      error_code:
-        type: "integer"
-        minimum: 0
-        format: "int32"
-        example: 1400
-      message:
-        type: "string"
-        pattern: "^[A-Za-z0-9 ]{1,64}$"
-        example: "Unexpected empty body."
\ No newline at end of file
+          type: integer
+          format: int32
+          example: 1400
+        message:
+          pattern: ^[A-Za-z0-9 ]{1,64}$
+          type: string
+          example: Unexpected empty body.
```

Comment:
I addressed that in https://github.com/immuni-app/immuni-documentation/pull/34/files#diff-4ebe1e340e131605718b39232a213acfR411 adding a `.` in  `pattern`.

I'll fix that here but this will probably require a rebase of #34

Reply:
Does it make sense to close one of the PRs and just leaving one?

Reply:
@gilbertotcc splitting the PR simplifies the editorial work of checking the equivalence of the spec after the  2->3 automatic conversion.

If you are in charge of the merging I will follow your guidelines though: just let me know.

Reply:
> I really like this PR that moves the API definition from Swagger to OpenAPI 3.
> 
> I've just found an error running the [Spectral](https://github.com/stoplightio/spectral) linter on this file: you can fix it.

You can try the online validator based on spectral https://teamdigitale.github.io/api-oas-checker/ that checks OAS spec with some of the Italian API Guidelines.

Reply:
Nope, I'm not in charge of merging. I was just asking. :smile:

Reply:
Ok :) So let's wait for the repo admin. Please, have a look at the https://teamdigitale.github.io/api-oas-checker/  and feel free to file issues on the linter there!

Reply:
Fixed in #134, thus resolving. Thanks. 🚀